Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -31,11 +31,9 @@ open class ChartDataSet: ChartBaseDataSet
         super.init()
     }
     
-    public override init(label: String?)
+    public override convenience init(label: String?)
```

Comment:
Why not do this in the ChartBaseDataSet?

Reply:
I think there is no obvious reason. `init(label: String?)` is right in ChartDataSet already.